src/Microsoft.AspNetCore.Server.Kestrel/Microsoft.AspNetCore.Server.Kestrel.csproj

```diff
@@ -12,6 +12,10 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Hosting" Version="$(AspNetCoreVersion)" />
```

Comment:
This is needed, right? The only class in this project, WebHostBuilderKestrelExtension, only seems to reference IWebHostBuilder, IServer and some config stuff. That should come from Hosting.Abstractions, Hosting.Server.Abstractions and Options respectively.

All of this should come in transitively. If that isn't the case, why is the Libuv project building which also has IWebHostBuilder extensions?